src/modules/sub-items/sub.items.module.js

```diff
@@ -386,20 +556,21 @@ export default class SubItemsModule extends Component {
         }
 
         const UniversalDiscovery = window.eZ.modules.UniversalDiscovery;
-        const { restInfo, parentLocationId, udwConfigBulkMoveItems } = this.props;
+        const { restInfo, parentLocationId, udwConfigBulkItems } = this.props;
         const { selectedItems } = this.state;
         const selectedItemsLocationsIds = [...selectedItems.values()].map(({ id }) => id);
-        const excludedMoveLocations = [parentLocationId, ...selectedItemsLocationsIds];
+        const excludedLocations = [parentLocationId, ...selectedItemsLocationsIds];
         const title = Translator.trans(/*@Desc("Choose location")*/ 'udw.choose_location.title', {}, 'sub_items');
         const udwProps = {
             title,
             restInfo,
             onCancel: this.closeUdw,
             onConfirm: this.onUdwConfirm,
             canSelectContent: ({ item }, callback) => {
-                callback(!excludedMoveLocations.includes(item.id));
+                callback(!excludedLocations.includes(item.id));
             },
-            ...udwConfigBulkMoveItems,
+            ...udwConfigBulkItems,
```

Comment:
Here should be passed config depends on the action type `udwConfigBulkMoveItems` or `udwConfigBulkAddLocation`